Services/DataBaseService/Startup.cs

```diff
@@ -30,6 +40,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 {
                     await context.Response.WriteAsync("Hello World!");
                 });
+
+                endpoints.MapGet("/1/", async context =>
```

Comment:
what is this for ?

Reply:
You're right, I forgot to remove this testing stuff